pkg/network/vmispec/network_test.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package vmispec_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+)
+
+var _ = Describe("Network", func() {
+	podNetwork := createPodNetwork("default")
+	multusDefaultNetwork := createMultusDefaultNetwork("network0", "default/nad0")
+	multusSecondaryNetwork1 := createMultusSecondaryNetwork("network1", "default/nad1")
+	multusSecondaryNetwork2 := createMultusSecondaryNetwork("network2", "default/nad2")
+	multusSecondaryNetwork3 := createMultusSecondaryNetwork("network3", "default/nad3")
+	DescribeTable("should return the only non default networks", func(inputNetworks, expectFilteredNetworks []v1.Network) {
```

Comment:
I miss the word `multus` in the description - i,e, `multus non-default networks`

Reply:
Done